sktime/transformations/panel/signature_based/_signature_method.py

```diff
@@ -62,7 +65,15 @@ def __init__(
         rescaling=None,
         sig_tfm="signature",
         depth=4,
+        backend="esig",
     ):
+        if backend == "esig":
+            _check_soft_dependencies("esig")
+        elif backend == "iisignature":
+            _check_soft_dependencies("iisignature")
+        else:
+            raise ValueError("backend must be one of 'esig' or 'iisignature'")
```

Comment:
not blocking - to improve this error message, you should refer to `SignatureTransformer`, as the error could be raised in a pipeline, and users would not know which component raised it.

Reply:
Good point, thanks. Done.